qztest/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+project(qztest)
+
+find_package(Qt5 REQUIRED COMPONENTS Network Test)
```

Comment:
What about tests for Qt4? `BUILD_WITH_QT4` implies that we work with Qt4 but tests are always using Qt5